upcloud/network_peering.go

```diff
@@ -0,0 +1,101 @@
+package upcloud
+
+import "encoding/json"
+
+type NetworkPeeringConfiguredStatus string
+
+type NetworkPeeringIPNetworkFamily string
+
+type NetworkPeeringState string
```

Comment:
Might be worthwhile to add a comment that describes how does that differ from `NetworkPeeringConfiguredStatus`. Most IDEs should display such comments so that would be a good hint for consumers

Reply:
Yes! I'll push changes.